tests/nvapi_d3d12.cpp

```diff
@@ -803,6 +804,10 @@ TEST_CASE("D3D12 methods succeed", "[.d3d12]") {
 
         auto e = ConfigureDefaultTestEnvironment(*dxgiFactory, *vulkan, *nvml, *lfx, adapter, output);
 
+        ALLOW_CALL(commandQueue, GetDevice(__uuidof(ID3DLowLatencyDevice), _))
+            .LR_SIDE_EFFECT(*_2 = static_cast<ID3DLowLatencyDevice*>(&lowLatencyDevice))
+            .LR_SIDE_EFFECT(lowLatencyDeviceRefCount++)
```

Comment:
While at it, I think our refcounts should be zero with your changes, so we could check this at the end of this section 

```
        REQUIRE(deviceRefCount == 0);
        REQUIRE(commandListRefCount == 0);
        REQUIRE(lowLatencyDeviceRefCount == 0);
```

Reply:
Like this? I'm not sure if I can just put such checks at the end of aggregate section or if they should be replicated inside each innermost section.

Reply:
Yes, from my understanding of Catch this should be good.